### PR TITLE
Add main page

### PR DESCRIPTION
**Pull request by ShinhyoBelok**

### CODE REVIEW

**Comment by ShinhyoBelok** [comment]
Add the main page with the mage "Hello Microverse!"

**Comment by MoRaad97** [comment]
Hi @ShinhyoBelok, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
 

- [ ] [OPTIONAL] It's best practice to Capitalized the first word of the commit   💪 for example "Add index.html file"

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MoRaad97 — COMMENTED** [review]
Hi @ShinhyoBelok, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
 

- [ ] [OPTIONAL] It's best practice to Capitalized the first word of the commit   💪 for example "Add index.html file"

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.main_message {
+  color: blue;
+}
```
Comment (ShinhyoBelok) [inline style.css:3]: end of the style file

**Pull request merged**